ms-ledger/routes/types.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+)
+
+type Controller struct {
```

Comment:
IMO the file name should be controller.go (types.go implies it is only type definitions.)